include/swift/Threading/Impl/chrono.h

```diff
@@ -0,0 +1,71 @@
+//===--- chrono.h - Utility functions for duration/time_point - -*- C++ -*-===//
```

Comment:
Does it make sense to rename this header to `chrono_utils.h` to match the namespace and to differentiate it from the C++ header `chrono`?